scripts/test_twemoji_mapping.py

```diff
@@ -54,16 +54,22 @@
 
     n = len(mapping)
     for i, emoji in enumerate(mapping, start=1):
-        emoji_surrogates = emoji["surrogates"]
-        name = emoji["primaryName"]
-        emoji = emojis.UnicodeEmoji.parse(emoji_surrogates)
-
-        if emoji.filename in known_files:
-            valid_emojis.append((emoji_surrogates, name))
-            print("[  OK  ]", f"{i}/{n}", name, *map(hex, map(ord, emoji_surrogates)), emoji.url)
-        else:
+        try:
+            emoji_surrogates = emoji["surrogates"]
+            name = emoji["primaryName"]
+            emoji = emojis.UnicodeEmoji.parse(emoji_surrogates)
+
+            if emoji.filename in known_files:
+                valid_emojis.append((emoji_surrogates, name))
+                print("[  OK  ]", f"{i}/{n}", name, *map(hex, map(ord, emoji_surrogates)), emoji.url)
+            else:
+                invalid_emojis.append((emoji_surrogates, name))
+                print("[ FAIL ]", f"{i}/{n}", name, *map(hex, map(ord, emoji_surrogates)), emoji.url)
+
+        except Exception as ex:
```

Comment:
surely we don't need this behaviour and any exception should just raise since with the current changes it won't be expected to raise anywhere